snow/consensus/snowman/poll/early_term_no_traversal.go

```diff
@@ -4,31 +4,137 @@
 package poll
 
 import (
+	"errors"
 	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/utils/bag"
 )
 
+var (
+	errPollDurationVectorMetrics = errors.New("failed to register poll_duration vector metrics")
+	errPollCountVectorMetrics    = errors.New("failed to register poll_count vector metrics")
+
+	terminationReason    = "reason"
+	exhaustedReason      = "exhausted"
+	earlyFailReason      = "early_fail"
+	earlyAlphaPrefReason = "early_alpha_pref"
+	earlyAlphaConfReason = "early_alpha_conf"
+
+	exhaustedLabel = prometheus.Labels{
+		terminationReason: exhaustedReason,
+	}
+	earlyFailLabel = prometheus.Labels{
+		terminationReason: earlyFailReason,
+	}
+	earlyAlphaPrefLabel = prometheus.Labels{
+		terminationReason: earlyAlphaPrefReason,
+	}
+	earlyAlphaConfLabel = prometheus.Labels{
+		terminationReason: earlyAlphaConfReason,
+	}
+)
+
+type earlyTermNoTraversalMetrics struct {
+	durPolls               *prometheus.GaugeVec
+	durExhaustedPolls      prometheus.Gauge
+	durEarlyFailPolls      prometheus.Gauge
+	durEarlyAlphaPrefPolls prometheus.Gauge
+	durEarlyAlphaConfPolls prometheus.Gauge
+
+	countPolls               *prometheus.CounterVec
+	countExhaustedPolls      prometheus.Counter
+	countEarlyFailPolls      prometheus.Counter
+	countEarlyAlphaPrefPolls prometheus.Counter
+	countEarlyAlphaConfPolls prometheus.Counter
```

Comment:
These variables are unused.
```suggestion
	durExhaustedPolls      prometheus.Gauge
	durEarlyFailPolls      prometheus.Gauge
	durEarlyAlphaPrefPolls prometheus.Gauge
	durEarlyAlphaConfPolls prometheus.Gauge

	countExhaustedPolls      prometheus.Counter
	countEarlyFailPolls      prometheus.Counter
	countEarlyAlphaPrefPolls prometheus.Counter
	countEarlyAlphaConfPolls prometheus.Counter
```

Reply:
updated